### PR TITLE
migrate to GitHub Actions

### PR DESCRIPTION
**Pull request by ocefpaf**
@bjlittle this should get the tests here working a bit faster and more reliably.

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+name: Tests
```
Comment (ocefpaf) [inline .github/workflows/tests.yml:1]: You need to have one GHA actions for it to start in PRs. I'm not sure if there is another way.

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        pip install -e . --force-reinstall
+
+    - name: Tests
+      run: |
+        coverage run setup.py test
+
+    # - name: Coveralls
+    #   uses: coverallsapp/github-action@master
+    #   with:
+    #     github-token: ${{ secrets.GITHUB_TOKEN }}
```
Comment (ocefpaf) [inline .github/workflows/tests.yml:34]: I left this one commented out b/c someone with admin privileges will need to set that token up. (I can remove this part of the code if you don't want it there.)

**Comment by bjlittle** [comment]
Closed by #66.

Thanks @ocefpaf. Stuck with `cirrus-ci` for continuity across SciTools repos.

I'm looking forward to the `cirrus-ci` vs GH actions discussion that's soon to come 👍

**Comment by bjlittle** [comment]
@ocefpaf To be fair, our move to `cirrus-ci` was the result of a need to quickly migrate off `travis-ci` as `iris` CI was effectively dead at the time.

I'm personally not vested in `cirrus-ci` as a solution, but it was a pattern being used for CI across SciTools... and I genuinely think that is up for debate.

I just opted for `cirrus-ci` here simply not to confuse matters, but I strongly suspect that we'll migrate to GH actions :+1:

**Pull request closed without merging**